osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -44,100 +39,165 @@ public ScoresContainer()
                 },
                 new FillFlowContainer
                 {
-                    Anchor = Anchor.TopCentre,
-                    Origin = Anchor.TopCentre,
-                    RelativeSizeAxes = Axes.X,
-                    AutoSizeAxes = Axes.Y,
-                    Width = 0.95f,
                     Direction = FillDirection.Vertical,
-                    Spacing = new Vector2(0, spacing),
-                    Margin = new MarginPadding { Vertical = spacing },
+                    AutoSizeAxes = Axes.Y,
+                    RelativeSizeAxes = Axes.X,
                     Children = new Drawable[]
                     {
-                        topScore = new DrawableTopScore(),
-                        scoreTable = new ScoreTable
+                        loadingContainer = new LoadingContainer
```

Comment:
code-wise and visually, this feels weird. it's pushing content down physically, which isn't what a loading spinner should do.